### DISCUSSION TITLE
What is "cd on quit" exactly ?

### DISCUSSION THREAD

**Discussion by lyndhurst** [jarun/nnn#1725] [Q&A] [answered]
Hi,

I am not sure if my config is working properly, because I am not sure what the feature is.
I read the man page and the doc, and, found it easy to configure with the `n` function added to my `rc` file. The problem is that I do not notice any changes in behavior.

So first, I would like to know what `cd on quit` is supposed to do. In my mind, it means that hitting `^G` would quit `nnn` to terminal prompt and `cd` to the directory `nnn` was last in.

Am I understanding the feature correctly ?

**Comment by N-R-K** [comment, marked as answer]
> Am I understanding the feature correctly ?

Yes, that's correct.

> The problem is that I do not notice any changes in behavior.

You need to actually launch with `n` as well.

**Reply by lyndhurst** [reply]
Thanks for your reply.
I was aware that I had to start it with `n` but after looking around, I had a conflicting bash_alias messing with the `n` function.

On top of that, I had the function on my `.profile` file. I moved it to my shell `rc` file for easier debugging, removed the alias, and it works just fine now.

I will have to move it back to `profile` to check if it still works, but I cannot log out because of a long running `sherd` command I do not want to interrupt.

Thanks again.
